src/Components/test/Microsoft.AspNetCore.Components.Build.Test/RazorIntegrationTestBase.cs

```diff
@@ -205,6 +230,30 @@ protected CompileToCSharpResult CompileToCSharp(string cshtmlRelativePath, strin
 
                 // Result of real code generation for the document under test
                 codeDocument = DesignTime ? projectEngine.ProcessDesignTime(projectItem) : projectEngine.Process(projectItem);
+
+                _output.Value.WriteLine("Use this output when opening an issue");
```

Comment:
I'm really proud of this 😁 

This produces the markdown that you see [here](https://github.com/aspnet/AspNetCore/issues/6187) as part of the test log. You can easily produce everything you need to log an issue from just a failing test.